setup.py

```diff
@@ -1,42 +1,24 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-
+import re
 
 try:
     from setuptools import setup
 except ImportError:
     from distutils.core import setup
-import os
-
-
-__version__ = "0.7"
-devStatus = "4 - Beta"  # default build status, see: https://pypi.python.org/pypi?%3Aaction=list_classifiers
-
-if "TRAVIS_BUILD_NUMBER" in os.environ and "TRAVIS_BRANCH" in os.environ:
-    print("This is TRAVIS-CI build")
-    print("TRAVIS_BUILD_NUMBER = {}".format(os.environ["TRAVIS_BUILD_NUMBER"]))
-    print("TRAVIS_BRANCH = {}".format(os.environ["TRAVIS_BRANCH"]))
 
-    __version__ += ".{}{}".format(
-        ""
-        if "release" in os.environ["TRAVIS_BRANCH"]
-        or os.environ["TRAVIS_BRANCH"] == "master"
-        else "dev",
-        os.environ["TRAVIS_BUILD_NUMBER"],
-    )
 
-    if (
-        "release" in os.environ["TRAVIS_BRANCH"]
-        or os.environ["TRAVIS_BRANCH"] == "master"
-    ):
-        devStatus = "5 - Production/Stable"
+with open("version.txt") as file:
+    __version__ = next(file).strip()
 
-    else:
-        devStatus = devStatus
+# just double check that version is correct
```

Comment:
Regular expression should have short examples comment
```
# Version must be either 0.1.1 or 0.1.dev1
assert re.match(r"^v\d\.\d\.\d$|^v\d\.\d\.\d\.dev\d+$", __version__)
```

Reply:
added